### PR TITLE
Address actions

### PR DESCRIPTION
**Pull request by sunilsabatp**
This PR handle the address actions (`delete, setPrimary, setBilling`).

### CODE REVIEW

**Review by mariahosfeld — COMMENTED** [review]
Functionality looks good to me.

File: public/static/locales/en/profile.json [hunk 1]
```diff
@@ -147,6 +147,12 @@
         "setAsPrimaryAddress": "Set as Primary Address",
         "setAsBillingAddress": "Set as Billing Address"
       },
+      "deleteAddress": "Delete Address",
+      "delete": "Delete",
+      "addressConfirmationMessage": "Are you sure you want to set this address as your {addressType, select, mailing {billing} other {{addressType}}} address? {isAddressSet, select, true {This will replace your current {addressType, select, mailing {billing} other {{addressType}}} address.} other {}}",
+      "deleteAddressConfirmationMessage": "Are you sure you want to delete this address? If you want to use it again, please add it as a new address.",
+      "confirm": "Confirm",
+      "address2": "Address 2(optional)",
```
Comment (mohitb35) [inline public/static/locales/en/profile.json:155]: Do some of the keys that are currently outside need to be moved/grouped?

For example, I see `delete`, and `address2` keys

File: src/features/user/Settings/EditProfile/AddressManagment/AddressDeleteModal.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      fetchUserAddresses();
+    } catch (error) {
+      setErrors(handleError(error as APIError));
+    } finally {
+      setIsModalOpen(false);
+      setIsLoading(false);
+    }
+  };
+  return (
+    <div className={styles.addrConfirmContainer}>
```
Comment (mohitb35) [inline src/features/user/Settings/EditProfile/AddressManagment/AddressDeleteModal.tsx:52]: I don't think `addrConfirmContainer` needs to be abbreviated.

File: src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss
Comment (mohitb35) [inline src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss:1]: Could you clean up this scss file? Spacing between classes is erratic

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@sunilsabatp Mostly minor feedback, except for the state management of `addressAction`

File: src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss
Comment (sunilsabatp) [reply src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss:1]: I will handle this in this PR 
image

Comment (sunilsabatp) [reply src/features/user/Settings/EditProfile/AddressManagment/AddressManagement.module.scss:1]: I'll cover this in this  https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2312

**Review by mohitb35 — APPROVED** [review]
@sunilsabatp Made a couple of small changes. LGTM.

File: public/static/locales/en/profile.json [hunk 3]
```diff
@@ -147,6 +147,12 @@
         "setAsPrimaryAddress": "Set as Primary Address",
         "setAsBillingAddress": "Set as Billing Address"
       },
+      "deleteAddress": "Delete Address",
+      "delete": "Delete",
+      "addressConfirmationMessage": "Are you sure you want to set this address as your {addressType, select, mailing {billing} other {{addressType}}} address? {isAddressSet, select, true {This will replace your current {addressType, select, mailing {billing} other {{addressType}}} address.} other {}}",
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply public/static/locales/en/profile.json]: Made this simpler in [035c34e](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2308/commits/035c34ed8d5326ecf0a48b0bfc79f5b0f5cb17e3)

**Pull request merged**